risc0/zkvm/src/exec/mod.rs

```diff
@@ -217,6 +217,7 @@ impl<'a> Executor<'a> {
         self.insn_counter = 0;
         self.segment_cycle = self.init_cycles;
         self.pre_pc = self.pc;
```

Comment:
Do we still need this field?

Reply:
ooo great catch! I think I can delete even more fields! I was even able to drop a clippy warning suppression in the `Session::new()` method.

Fixed in: 56ae572c776e00d4dfe9d1631d71bfc41a2f883c